tests_disabled.json

```diff
@@ -0,0 +1,21 @@
+{
+    "disable" : true,
+    "disabled" : [
+        "regression.test_stress_pipeline.PipelineFaultInjection",
```

Comment:
A good description is required, why the test fails and what is the issue number for the test. If issue number is not provided for the disabled test, then the developers don't know about the issue and it will never be fixed.